apps/docs/stories/components/Summary/Docs.mdx

```diff
@@ -0,0 +1,44 @@
+import { Meta, Title, ArgTypes, Source } from '@storybook/blocks';
+import { filterDocsByTag, ExportedParts } from '../../utils';
+import { codeExampleFull } from './example';
+import { ComponentBox } from '../../utils/documentation-components';
+import * as JustifiCheckoutSummary from './index.stories';
+import { setUpMocks } from '../../utils/mockAllServices';
+
+{setUpMocks()}
+
+<Meta
+  title="Modular Checkout/Checkout Summary"
+  Components="justifi-checkout-summary"
+  of={JustifiCheckoutSummary}
+/>
+
+<Title />
+
+---
+
+Renders a summary of checkout details as part of a checkout flow.
+
+This component is **designed to be used within** the `justifi-modular-checkout` and **does not accept props directly**. Instead, it relies on shared state passed through the Stencil Store, managed by the wrapper component.
```

Comment:
Again update to '... on the shared state...' and '... managed by the `justifi-modular-checkout` component.'